src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -228,15 +222,24 @@ class ContextPickerModal extends React.Component<Props> {
       .filter(({status}) => status.id !== 'pending_deletion')
       .map(({slug}) => ({label: slug, value: slug}));
 
+    const customStyles = {
+      option: provided => ({
+        ...provided,
+        borderTop: `1px solid ${theme.gray1}`,
```

Comment:
Why the one off style here?

Reply:
If this is also to match @ckj's design, we should maybe implement this in other select boxes?

I think visually this is a little wonky too